### PR TITLE
Small output changes

### PR DESCRIPTION
**Pull request by aristosgi**
I made a few small changes to the output message to make it more friendly for the users.The difference is shown in the screenshots below

Screenshot 2022-03-23 at 10 44 35 PM
Screenshot 2022-03-23 at 10 44 19 PM

### CODE REVIEW

**Review by aristosgi — COMMENTED** [review]
In this files there are no changes.

File: sherlock/notify.py [hunk 1]
```diff
@@ -155,6 +155,33 @@ def start(self, message):
               Fore.GREEN + f"] {title}" +
               Fore.WHITE + f" {message}" +
               Fore.GREEN + " on:")
+        # An empty line between first line and the result(more clear output)
+        print('\r')
+
+        return
+
+    def finish(self, message="Results have been uploaded"):
```
Comment (TheYahya) [inline sherlock/notify.py:163]: We don't upload anything. so it's better if have a better sentence here. 
like: `The processing has been finished.`

Comment (aristosgi) [reply sherlock/notify.py]: I change it and also changes finish to end. Looks better to me if you dont agree i make it 'finish' again

**Review by TheYahya — APPROVED** [review]

File: sherlock/notify.py [hunk 2]
```diff
@@ -155,6 +155,34 @@ def start(self, message):
               Fore.GREEN + f"] {title}" +
               Fore.WHITE + f" {message}" +
               Fore.GREEN + " on:")
+        # An empty line between first line and the result(more clear output)
+        print('\r')
+
+        return
+
+    def finish(self, message="The processing has been finished."):
```
Comment (aristosgi) [inline sherlock/notify.py:163]: Small changes in finish method to modify the output of the results

File: sherlock/notify.py [hunk 3]
```diff
@@ -160,7 +160,7 @@ def start(self, message):
 
         return
 
-    def finish(self, message="Results have been uploaded"):
+    def finish(self, message="The processing has been finished."):
```
Comment (aristosgi) [inline sherlock/notify.py:163]: modify the final output sentence

**Pull request merged**